akka-cluster/src/main/scala/akka/cluster/ClusterReadView.scala

```diff
@@ -109,12 +109,12 @@ private[akka] class ClusterReadView(cluster: Cluster) extends Closeable {
   def status: MemberStatus = self.status
 
   /**
-   * Is this node the leader?
+   * Is this node the leader (if using cluster teams the leader of the team of this node)?
    */
   def isLeader: Boolean = leader.contains(selfAddress)
 
   /**
-   * Get the address of the current leader.
+   * Get the address of the current leader (if using cluster teams the leader of the team of this node).
```

Comment:
A given node is always part of some team, right? Therefore I suggest to rephrase this to:

> Get the address of the current team leader